translation-tests/src/integration-test/java/eu/europeana/api/translation/tests/BaseTranslationTest.java

```diff
@@ -125,6 +126,8 @@ static void setProperties(DynamicPropertyRegistry registry) {
 
     registry.add("translation.google.projectId", () -> "project-id-test");
     registry.add("translation.google.usehttpclient", () -> "true");
+    registry.add("spring.redis.host", () -> "localhost");
```

Comment:
should use a single property for redis connection url, see also the comments on the properties file